### PR TITLE
Print testfile content for debugggin

### PR DESCRIPTION
**Pull request by nicolasbock**

### CODE REVIEW

**Comment by dosaboy** [comment]
i'm not really in favour of this as it will spam the logs with a huge amount of data that we could get by looking at the template file itself.

**Comment by nicolasbock** [comment]
This change helped me in really making sure that I got the scenario input correct. This change adds 3912 lines to the output and makes is 6.4% larger. That's not too bad 😉 

```console
wc -l main.output debug_testfile.output 
   61367 main.output
   65279 debug_testfile.output
  126646 total
```

**Comment by dosaboy** [comment]
another option would be to add a new envvar similar to ```TESTS_LOG_LEVEL_DEBUG``` so that this can be enabled optionally.

**Comment by nicolasbock** [comment]
Yes, that would work as well. Let me work on this.

**Review by dosaboy — COMMENTED** [review]
Looks good. Commit msg could do with a tidy up and maybe also a comment about the new env var.

**Comment by nicolasbock** [comment]
> Looks good. Commit msg could do with a tidy up and maybe also a comment about the new env var.

Done.

File: tests/unit/utils.py [hunk 1]
```diff
@@ -355,12 +355,19 @@ def create_files_inner2(*args, **kwargs):
...
 
+                log_artifacts = os.environ.get('TESTS_LOG_TEST_ARTIFACTS',
+                                               'no') == 'yes'
+
                 for path, content in files_to_create.items():
                     path = os.path.join(dtmp, path)
                     if not os.path.exists(os.path.dirname(path)):
                         os.makedirs(os.path.dirname(path))
 
-                    log.debug("creating test file %s", path)
```
[the remark this answers is not in the document]
Comment (nicolasbock) [reply tests/unit/utils.py:363]: done

**Pull request merged**